helpers/calendar/analytics_helper.py

```diff
@@ -94,48 +94,57 @@ def get_calendar_id_name(self, query):
                 raise JsonError(Message='No rooms in this location')
             else:
                 raise GraphQLError("No rooms in this location")
-        result = [{'name': room.name, 'calendar_id': room.calendar_id}
-                  for room in rooms_in_locations.all()]
+        result = [{
+            'name': room.name,
+            'room_id': room.id,
+            'calendar_id': room.calendar_id
+        }for room in rooms_in_locations.all()]
         return result
 
-    def get_all_events_in_a_room(self, calendar_id, min_limit, max_limit):
+    def get_all_events_in_a_room(self,
+                                 room_id,
+                                 event_start_time,
+                                 event_end_time):
         """ Get all events in a room
          :params
-            - calendar_id - for specific room
-            - min_limit, max_limit(Time range)
+            - room_id - for specific room
+            - event_start_time, event_end_time(Time range)
         """
-        try:
-            events_result = get_google_calendar_events(
-                calendarId=calendar_id, timeMin=min_limit,
-                timeMax=max_limit, singleEvents=True,
-                orderBy='startTime')
-        except Exception:
-            raise GraphQLError("Resource not found")
-        all_events = events_result.get('items', [])
-        return all_events
+        events = EventsModel.query.all()
+        result = []
+        for event in events:
+            if event.room_id == room_id:
+                room_event = {
+                    'room_id': event.room_id,
+                    'event_start_time': event.start_time,
+                    'event_end_time': event.end_time,
+                    'event_title': event.event_title,
+                    'participants': event.number_of_participants,
+                    'event_id': event.event_id,
+                }
+                result.append(room_event)
+            if event.room_id == room_id and \
```

Comment:
I believe you can do away with the `\` @nakatuddesuzan

Reply:
No, I can't @ThaDeveloper , it is necessary for breaking the line, or else it will throw an error. Kindly try it out on your end and see.

Reply:
The break is necessary to avoid flake8 issues

Reply:
You can format it like this without any flake8 violation
![image](https://user-images.githubusercontent.com/12385482/54809570-b9590300-4c94-11e9-8b66-fbe126c0f71e.png)

Reply:
Okay, thanks, this has been worked on